### PR TITLE
Add Contrast Filter

### PR DESCRIPTION
**Pull request by sarthak-saxena**
- [x] **Tests**

### CODE REVIEW

**Comment by nicoja** [comment]
Thanks to manage it ;)
Best Regards

File: src/filters/Contrast.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      blue = (blue < 0 ? 0 : (blue > 255 ? 255 : blue));
+
+      data[i] = red;
+      data[i + 1] = green;
+      data[i + 2] = blue;
+    }
+  };
+
+  /**
+   * get/set filter contrast.  The contrast is a number between -100 and 100.&nbsp;
```
Comment (albertogasparin) [inline src/filters/Contrast.js]: Typo in the comment `&nbsp;`

**Comment by lavrton** [comment]
Thanks for a very high-quality PR.

**Comment by sarthak-saxena** [comment]
@lavrton when are you planning to update the package on npm?

**Comment by lavrton** [comment]
@sarthak-saxena not sure yet. I want to finish the new `Transformer` tool first.

But you can always install from the master branch or a particular commit.

**Pull request merged**